console/src/app/app.component.html

```diff
@@ -28,7 +28,7 @@
                     {{temporg?.name ? temporg.name : 'NO NAME'}}
                 </button>
 
-                <ng-template appHasRole [appHasRole]="['iam.write']">
+                <ng-template appHasRole [appHasRole]="['org.create']">
```

Comment:
its not the same, some have iam.write (iam manager is able to create new organisation with a new user as org owner), and some have org.create (org manager is able to create new org with himself as owner)

Reply:
replaced with (org.create)?(iam.write)?